src/components/editor/editor-pane/tool-bar/editor-preferences/editor-preference-ligatures-select.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React, { ChangeEvent, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../../../redux'
+import { setEditorLigatures } from '../../../../../redux/editor/methods'
+import { EditorPreferenceInput, EditorPreferenceInputType } from './editor-preference-input'
+
+export const EditorPreferenceLigaturesSelect: React.FC = () => {
+  const ligaturesEnabled = useSelector((state: ApplicationState) => Boolean(state.editorConfig.ligatures).toString())
+  const saveLigatures = useCallback((event: ChangeEvent<HTMLSelectElement>) => {
+    const ligaturesActivated: boolean = event.target.value === 'true'
+    setEditorLigatures(ligaturesActivated)
+  }, [])
+  const { t } = useTranslation()
+
+  return (
+    <EditorPreferenceInput onChange={saveLigatures} value={ligaturesEnabled} property={"ligatures"}
+                           type={EditorPreferenceInputType.BOOLEAN}>
+      <option value='false'>{t(`editor.modal.preferences.ligatures.off`)}</option>
+      <option value='true'>{t(`editor.modal.preferences.ligatures.on`)}</option>
+    </EditorPreferenceInput>
```

Comment:
Why don't you use an `EditorPreferenceBooleanProperty` instead?

Reply:
Because the property components don't offer an "onChange". They save the result into the redux state. I think this is okay until we add the spell check.

Reply:
Hm, maybe we should add an optional `onChange`-callback`?
But if you just want to do it like this, it's okay for me too.

Reply:
I don't think that it's worth here, because this is only a placeholder until the type checker is built.